src/kakarot/instructions.cairo

```diff
@@ -28,6 +28,12 @@ from kakarot.instructions.block_information import BlockInformation
 from kakarot.instructions.system_operations import SystemOperations
 from kakarot.instructions.sha3 import Sha3
 
+// Packed Felt implementation
```

Comment:
not sure we want to comment the reason why we do some imports, at least somehow non standard